src/components/action-popover/action-popover.style.ts

```diff
@@ -19,12 +20,134 @@ const Menu = styled.div`
     `${theme.zIndex?.popover}`}; // TODO (tokens): implement elevation tokens - FE-4437
 `;
 
+type variants = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+
+function getLeftPaddingValues(variant: variants) {
```

Comment:
**comment** - really just repeating what I already put in the parent component: I'm not a fan of using a single numeric `variant` prop to calculate left and right padding and so on. I'd rather calculate what we want in the parent component and pass appropriate padding (etc) props to the styled component.

Reply:
Agreed, if we pass props to these styled components, we can do css applications for each one, the styled component might get a bit long, but I agree its a much better approach 👍

Reply:
I agree with Robin, I also think as some of the variants produce the same value for right/ left padding they don't need to be separate in whatever you go for in the end. For example variants 3 and 4 produce the same padding results so you probably don't need both etc.